src/styles/components/_specification-block.scss

```diff
@@ -3,14 +3,54 @@ $breakpoint-tablet: 820px;
 .cmp-specifications-block {
   &__wrapper {
     display: block;
-
-    @media only screen and (min-width: $breakpoint-tablet) {
-      display: grid;
-      grid-template-columns: 1fr 1fr;
-    }
+    padding: 3.75rem 1.25rem; // top&bottom: 60px/16; left&right: 20px/16
   }
 
   &__heading {
     font-size: map.get($font-sizes, "md");
+    font-family: Viga, sans-serif;
+  }
+
+  &__content {
+    font-size: map.get($font-sizes, "sm"); // 1.25rem, 20px/16
+  }
+
+  @media only screen and (min-width: $breakpoint-tablet) {
+    // grid of data fields & content (.cmp-specifications-list)
+    margin: 3.125rem 8.4375rem; // 50px/16 top & bottom; 135px/16 left & right
```

Comment:
The gap for the grid elements is much larger than what figma shows it should be. Try removing line 23, and make the margin on line 20 `margin: auto 8.4375rem;` to make them closer together.

Reply:
I see what you mean now. I fixed up several things: 

- I should have had margin instead of padding for the `&__wrapper`
- I needed to get rid of the default margin around the `&__heading`
- Then in the media query, I needed to get rid of the margin at larger screen sizes. (to counteract some of the grid gaps)

How does it look now? https://github.com/sparkbox/accessible-components/pull/46/commits/2b8677dcd30d06376d4245f4281f3c39a898a0ae

Reply:
This looks much better 🏆